_pages/docs.html

```diff
@@ -7,26 +7,42 @@
 
 <div class="container">
 
-<div class="container mb-2">
-<div class="media">
-  <a href="https://github.com/heliophysicsPy/standards/blob/master/standards.md"><i class="fas fa-book fa-2x mr-3"></i></a>
-  <div class="media-body">
-    <h5 class="mt-0">PyHC Standards</h5>
-    In order to foster an open, welcoming, collaborative, and interoperable environment, this document lists a set of standards for the Python in Heliophysics Community.<br>
-    <a href="https://doi.org/10.5281/zenodo.2529131"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.2529131.svg" alt="DOI"></a>
+  <div class="container mb-2">
+    <div class="media">
+      <a href="https://github.com/heliophysicsPy/standards/blob/master/standards.md"><i class="fas fa-book fa-2x mr-3"></i></a>
+      <div class="media-body">
+        <h5 class="mt-0">PyHC Standards</h5>
+        In order to foster an open, welcoming, collaborative, and interoperable environment, this document lists a set of standards for the Python in Heliophysics Community.<br>
+        <a href="https://doi.org/10.5281/zenodo.2529131"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.2529131.svg" alt="DOI"></a>
+      </div>
+    </div>
   </div>
-</div>
-</div>
 
-<div class="container mb-2">
-<div class="media">
-  <a href="https://zenodo.org/record/2537188#.XOXWVC2ZN24"><i class="fas fa-book fa-2x mr-3"></i></a>
-  <div class="media-body">
-    <h5 class="mt-0">2018 PyHC Meeting Report</h5>
-    The official meeting report from the 2018 community meeting.<br>
-    <a href="https://doi.org/10.5281/zenodo.2537188"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.2537188.svg" alt="DOI"></a>
+  <br>
+
+  <div class="container mb-2">
+    <div class="media">
+      <a href="https://zenodo.org/record/2537188#.XOXWVC2ZN24"><i class="fas fa-book fa-2x mr-3"></i></a>
+      <div class="media-body">
+        <h5 class="mt-0">2018 PyHC Meeting Report</h5>
+        The official meeting report from the 2018 community meeting.<br>
+        <a href="https://doi.org/10.5281/zenodo.2537188"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.2537188.svg" alt="DOI"></a>
+      </div>
+    </div>
+  </div>
+
+  <br>
+
+  <div class="container mb-2">
+    <div class="media">
+      <a href="{{ site.baseurl }}/docs/reports/"><i class="fas fa-book fa-2x mr-3"></i></a>
+      <div class="media-body">
+        <h5 class="mt-0">
+          <a href="{{ site.baseurl }}/docs/reports/">PyHC Quarterly Reports</a>
+        </h5>
+        Quarterly reports on activity within the Python in Heliophysics Community.<br>
+      </div>
+    </div>
```

Comment:
I wish this diff hadn't rendered so terribly. This div for the quarterly reports link is the only thing I added. The changes above were just improving indentation. For some reason GitHub wants to make my indentation changes look like I blew away everything and redid it. I did not.

Reply:
Not a problem, makes sense what you did! :)